packages/FreiFahren_BE-NLP/main.py

```diff
@@ -81,10 +79,12 @@ def process_new_message(timestamp, message):
                 info.get('direction'),
             )
     else:
-        print('No valuable information found')
+        logger.info('No info found')
```

Comment:
Please make this more precise, like where this Log has been made and what exact information has not been found